.github/pr-notify.yml

```diff
@@ -0,0 +1,35 @@
+name: Checkbox Notify
+
+on:
+  pull_request:
+    types: [opened, synchronize]
```

Comment:
We don't want to run on synchronize otherwise it might notify on subsequent commits